steering_vectors/aggregators.py

```diff
@@ -1,10 +1,33 @@
-from typing import Callable
+from typing import Callable, Union
 
+from sklearn.linear_model import LinearRegression, LogisticRegression
 from torch import Tensor
 import torch
+import torch.nn.functional as F
 
 
 Aggregator = Callable[[Tensor, Tensor], Tensor]
+Regression = Union[LinearRegression, LogisticRegression]
+
+
+@torch.no_grad()
+def linear_aggregator(pos_acts: Tensor, neg_acts: Tensor) -> Tensor:
+    """
+    An aggregator that uses linear regression to calculate a steering vector.
+    """
+    return _get_normalized_regression_coef(
+        pos_acts, neg_acts, LinearRegression(fit_intercept=False)
+    )
+
+
+@torch.no_grad()
+def logistic_aggregator(pos_acts: Tensor, neg_acts: Tensor) -> Tensor:
+    """
+    An aggregator that uses logistic regression to calculate a steering vector.
+    """
+    return _get_normalized_regression_coef(
+        pos_acts, neg_acts, LogisticRegression(fit_intercept=False)
```

Comment:
It looks like Scikit learn has a bunch of options to customize the LogisticRegression calculation: https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html. Would it make sense to let the user control some of these? If so, we could have the `logistic_aggregator` be a function which returns an `Aggregator`, rather than being an `Aggregator` itself.

The downside would be that this would mean invoking this aggregator would be slightly different than the other pre-built aggregators. That said, we could potentially make all the pre-built aggregators into functions which return aggregators rather than being raw aggregators themselves. That would future-proof us against wanting to add params to the other built-in aggregators in the future too. :thinking:

What do you think? cc @dtch1997

Reply:
> If so, we could have the logistic_aggregator be a function which returns an Aggregator, rather than being an Aggregator itself.

I was thinking of using a pattern like this, but decided against it to make all aggregators behave similarly. 

> we could potentially make all the pre-built aggregators into functions which return aggregators rather than being raw aggregators themselves. That would future-proof us against wanting to add params to the other built-in aggregators in the future too.

I like this approach. If you decide in favor of this, I'd be happy to do the refactoring as part of this PR.

Reply:
👍 Yeah I'd vote to just make everything a function which returns an `Aggregator`